scripts/release.sh

```diff
@@ -281,6 +281,8 @@ main() {
       maybe_run docker push "quay.io/coreos/etcd:${RELEASE_VERSION}-${TARGET_ARCH}"
       log_callout "Pushing container images to gcr.io ${RELEASE_VERSION}-${TARGET_ARCH}"
       maybe_run docker push "gcr.io/etcd-development/etcd:${RELEASE_VERSION}-${TARGET_ARCH}"
+      log_callout "Pushing container images to artifact registry${RELEASE_VERSION}-${TARGET_ARCH}"
```

Comment:
I think you missed a space here

```suggestion
log_callout "Pushing container images to artifact registry ${RELEASE_VERSION}-${TARGET_ARCH}"
```

Reply:
Good spotting, fixed thanks!